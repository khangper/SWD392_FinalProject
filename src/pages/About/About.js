import React from "react";
import Header from "../../components/Header/Header";
import "./About.css";
import { Link } from "react-router-dom";
import { CiMobile1 } from "react-icons/ci";
import { GoPeople } from "react-icons/go";
import { LiaCertificateSolid } from "react-icons/lia";
import { SlGlobe } from "react-icons/sl";
import Footer from "../../components/Footer/Footer";

const About = () => {
  return (
    <div className="about">
      <Header />
      <main>
        <div className="about-background-parent">
          <div className="about-background">
            <div className="about-about-links">
              <div className="about-link">
                <a href="/about-us" className="about-about1">
                  About
                </a>
              </div>
              <div className="about-other-links">
                <div className="about-other-link-items">
                  {/* <a href="/about-us" className="about-about1">
                  About
                </a> */}

                  <a href="/our-blog" className="about-blog">
                    Blog
                  </a>

                  <a href="/company-details" className="about-company">
                    Company
                  </a>
                  <a href="/career" className="about-careers">
                    Careers
                  </a>
                  <a href="/press" className="about-press">
                    Press
                  </a>
                </div>
              </div>
            </div>
            <div className="about-learning-heading">
              <h2 className="about-heading-21">
                Improving Lives Through Learning
              </h2>
            </div>
          </div>

          <div className="about-features-content-wrapper">
            <div className="about-features-content">
              <div className="about-features-wrapper">
                <div className="about-features-inner-wrapper">
                  <div className="about-heading-2-our-features">
                    <h2 className="about-heading-22">Our Features</h2>
                  </div>
                  <div className="about-on-cursus-parent">
                    <div className="about-on-cursus">
                      On Cursus, you have access to:
                    </div>
                    <div className="about-line-wrapper"></div>
                  </div>
                </div>
              </div>
              <div className="about-feature-list">
                <div className="about-feature-items">
                  <CiMobile1 className="about-feature-icons" />
                  <div className="about-feature-descriptions">
                    <div className="about-heading-4-wrapper">
                      <div className="about-heading-4">Mobile learning</div>
                    </div>
                    <div className="about-container">
                      <p className="about-content">
                        Quisque nec volutpat sem. Interdum et malesuada fames ac
                        ante ipsum primis in faucibus.
                      </p>
                    </div>
                  </div>
                </div>

                <div className="about-feature-items">
                  <GoPeople className="about-feature-icons" />
                  <div className="about-feature-descriptions">
                    <div className="about-heading-4-wrapper">
                      <div className="about-heading-41">
                        Academic & Technical Support
                      </div>
                    </div>
                    <div className="about-container">
                      <p className="about-content">
                        Quisque nec volutpat sem. Interdum et malesuada fames ac
                        ante ipsum primis in faucibus.
                      </p>
                    </div>
                  </div>
                </div>

                <div className="about-feature-items">
                  <LiaCertificateSolid className="about-feature-icons" />
                  <div className="about-feature-descriptions">
                    <div className="about-heading-4-wrapper">
                      <div className="about-heading-42">
                        Sharable Certificates
                      </div>
                    </div>
                    <div className="about-container">
                      <p className="about-content">
                        Quisque nec volutpat sem. Interdum et malesuada fames ac
                        ante ipsum primis in faucibus.
                      </p>
                    </div>
                  </div>
                </div>

                <div className="about-feature-items">
                  <SlGlobe className="about-feature-icons" />
                  <div className="about-feature-descriptions">
                    <div className="about-heading-4-wrapper">
                      <div className="about-heading-43">
                        An Inclusive Experience
                      </div>
                    </div>
                    <div className="about-container">
                      <p className="about-content">
                        Quisque nec volutpat sem. Interdum et malesuada fames ac
                        ante ipsum primis in faucibus.
                      </p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>

          <div className="about-our-story">
            <div className="about-story-content">
              <div className="about-story-wrapper">
                <div className="about-heading-2-our-story">
                  <h2 className="about-heading-23">Our Story</h2>
                  <div className="about-line-story"></div>
                </div>
                <div className="about-our-story-container">
                  <p className="about-our-story-content">
                    Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed
                    consectetur vel dolor id ultrices. Proin a magna at mi
                    pretium pulvinar in eu enim. Nulla vel lacus lectus. Donec
                    at venenatis augue. Nam vitae purus placerat, hendrerit nisl
                    id, finibus magna. Etiam pharetra gravida ornare. Donec
                    sagittis, ipsum in egestas egestas, dui lorem sollicitudin
                    justo, ut ullamcorper velit neque eu velit. Ut et fringilla
                    elit. Mauris augue augue, auctor a blandit ac, convallis
                    eget neque. Curabitur in ante ante. Nullam laoreet tempus
                    erat at ornare. In nisl nisi, dapibus eget facilisis sit
                    amet, commodo quis nibh.
                  </p>
                </div>
              </div>
            </div>
            <img
              src="https://gambolthemes.net/html-items/cursus-new-demo/images/about/stroy_img.png"
              alt=""
              className="about-story-img"
            />
          </div>

          <div className="about-features-content-wrapper-our-global">
            <div className="about-features-content">
              <div className="about-features-wrapper">
                <div className="about-features-inner-wrapper">
                  <div className="about-heading-2-our-features">
                    <h2 className="about-heading-22">Our Global Reach</h2>
                  </div>
                  <div className="about-on-cursus-parent">
                    <div className="about-on-cursus">
                      Cursus is the leading global marketplace for teaching and
                      learning, connecting millions of students to the skills
                      they need to succeed.
                    </div>
                    <div className="about-line-wrapper"></div>
                  </div>
                </div>
              </div>
              <div className="about-feature-list-our-global">
                <div className="about-feature-items-our-global">
                  <div className="about-feature-descriptions-our-global">
                    <div className="about-heading-5-wrapper-our-global">
                      <h1 className="about-heading-5">25K</h1>
                    </div>
                    <div className="about-container-our-global">
                      <p className="about-content-our-global">Instructors</p>
                    </div>
                  </div>
                </div>

                <div className="about-feature-items-our-global">
                  <div className="about-feature-descriptions-our-global">
                    <div className="about-heading-5-wrapper-our-global">
                      <h1 className="about-heading-5">95K</h1>
                    </div>
                    <div className="about-container-our-global">
                      <p className="about-content-our-global">Course</p>
                    </div>
                  </div>
                </div>

                <div className="about-feature-items-our-global">
                  <div className="about-feature-descriptions-our-global">
                    <div className="about-heading-5-wrapper-our-global">
                      <h1 className="about-heading-5">40M</h1>
                    </div>
                    <div className="about-container-our-global">
                      <p className="about-content-our-global">
                        Course enrollments
                      </p>
                    </div>
                  </div>
                </div>

                <div className="about-feature-items-our-global">
                  <div className="about-feature-descriptions-our-global">
                    <div className="about-heading-5-wrapper-our-global">
                      <h1 className="about-heading-5">50+</h1>
                    </div>
                    <div className="about-container-our-global">
                      <p className="about-content-our-global">Languages</p>
                    </div>
                  </div>
                </div>

                <div className="about-feature-items-our-global">
                  <div className="about-feature-descriptions-our-global">
                    <div className="about-heading-5-wrapper-our-global">
                      <h1 className="about-heading-5">595+</h1>
                    </div>
                    <div className="about-container-our-global">
                      <p className="about-content-our-global">
                        Membership Partner
                      </p>
                    </div>
                  </div>
                </div>

                <div className="about-feature-items-our-global">
                  <div className="about-feature-descriptions-our-global">
                    <div className="about-heading-5-wrapper-our-global">
                      <h1 className="about-heading-5">295</h1>
                    </div>
                    <div className="about-container-our-global">
                      <p className="about-content-our-global">Countries</p>
                    </div>
                  </div>
                </div>
              </div>
            </div>
          </div>

          <div className="about-meet-our-team">
            <div className="about-team-content">
              <div className="about-team-wrapper">
                <div className="about-heading-6-wrapper">
                  <h2 className="about-heading-6">Meet Our Team</h2>
                </div>
                <div className="about-meet-parent">
                  <div className="about-meet-container">
                    <p className="about-meet-content">
                      A perfect blend of creativity and technical wizardry. The
                      best people formula for great websites!
                    </p>
                  </div>
                  <div className="about-line-wrappers"></div>
                </div>
              </div>
            </div>
            <div className="about-team-call-to-action-parent">
              <div className="about-team-call-to-action">
                <div className="about-team-call-container">
                  <p className="about-team-call-content">
                    Morbi eget elit eget turpis varius mollis eget vel massa.
                    Donec porttitor, sapien eget commodo vulputate, erat felis
                    aliquam dolor, non condimentum libero dolor vel ipsum. Sed
                    porttitor nisi eget nulla ullamcorper eleifend. Fusce
                    tristique sapien nisi, vel feugiat neque luctus sit amet.
                    Quisque consequat quis turpis in mattis. Maecenas eget
                    mollis nisl. Cras porta dapibus est, quis malesuada ex
                    iaculis at. Vestibulum egestas tortor in urna tempor, in
                    fermentum lectus bibendum. In leo leo, bibendum at pharetra
                    at, tincidunt in nulla. In vel malesuada nulla, sed
                    tincidunt neque. Phasellus at massa vel sem aliquet sodales
                    non in magna. Ut tempus ipsum sagittis neque cursus euismod.
                    Vivamus luctus elementum tortor, ac aliquet dolor vehicula
                    et.
                  </p>
                </div>
            
                <button className="about-button-join-our-team">
                  Join Our Team
                </button>
                
              </div>
              <img
                src="https://gambolthemes.net/html-items/cursus-new-demo/images/about/team.jpg"
                alt=""
                className="about-team-img"
              />
            </div>
          </div>

        </div>
      </main>

      <Footer/>
    </div>
  );
};

export default About;
