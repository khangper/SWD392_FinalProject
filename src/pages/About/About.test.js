import React from "react";
import { render, screen } from "@testing-library/react";
import About from "../About/About"
import "@testing-library/jest-dom/extend-expect";
import { renderWithProviders } from "../../constant/test-util";

describe("About", () => {
  beforeAll(() => {
    jest.spyOn(console, "error").mockImplementation(() => {});
    jest.spyOn(console, "log").mockImplementation(() => {});
  });

  afterAll(() => {
    console.error.mockRestore();
    console.log.mockRestore();
  });
  it("renders the About component", async () => {
    const { container } = renderWithProviders(<About />);
    expect(container).toBeDefined();
    expect(container).toMatchSnapshot();
  });
});