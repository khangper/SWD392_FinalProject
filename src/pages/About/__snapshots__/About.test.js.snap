// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`About renders the About component 1`] = `
<div>
  <div
    class="about"
  >
    <header
      class="header"
    >
      <a
        href="/home"
      >
        <button
          class="header-link1"
        >
          Back To Cursus
        </button>
      </a>
      <a
        href="/home"
      >
        <img
          alt=""
          class="header-group-icon"
          src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo.svg"
        />
      </a>
      <div
        class="profile"
      >
        <img
          alt=""
          class="header-avatar"
          src="profile-img.jpg"
        />
      </div>
    </header>
    <main>
      <div
        class="about-background-parent"
      >
        <div
          class="about-background"
        >
          <div
            class="about-navigation-links-parent"
          >
            <div
              class="about-navigation-links"
            >
              <a
                class="about-about"
                href="/about-us"
              >
                About
              </a>
            </div>
            <div
              class="about-navigation-links-1"
            >
              <a
                class="about-blog"
                href="/our-blog"
              >
                Blog
              </a>
            </div>
            <div
              class="about-navigation-links-2"
            >
              <a
                class="about-company"
                href="/company-details"
              >
                Company
              </a>
            </div>
            <div
              class="about-navigation-links-3"
            >
              <a
                class="about-careers"
                href="/career"
              >
                Careers
              </a>
            </div>
            <div
              class="about-navigation-links-4"
            >
              <a
                class="about-press"
                href="/press"
              >
                Press
              </a>
            </div>
          </div>
          <div
            class="about-learning-heading"
          >
            <h2
              class="about-heading-2"
            >
              Insights, ideas, and stories
            </h2>
          </div>
        </div>
        <div
          class="about-features-content-wrapper"
        >
          <div
            class="about-features-content"
          >
            <div
              class="about-features-wrapper"
            >
              <div
                class="about-features-inner-wrapper"
              >
                <div
                  class="about-heading-2-our-features"
                >
                  <h3
                    class="about-heading-22"
                  >
                    Our Features
                  </h3>
                </div>
                <div
                  class="about-on-cursus-parent"
                >
                  <div
                    class="about-on-cursus"
                  >
                    On Cursus, you have access to:
                  </div>
                  <div
                    class="about-line-wrapper"
                  />
                </div>
              </div>
            </div>
            <div
              class="about-feature-list"
            >
              <div
                class="about-feature-items"
              >
                <div
                  class="about-icon-parent"
                >
                  <svg
                    class="about-feature-icons"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <g
                      id="Mobile_1"
                    >
                      <g>
                        <path
                          d="M10,18.933h4a.5.5,0,0,0,0-1H10a.5.5,0,0,0,0,1Z"
                        />
                        <path
                          d="M16.727,21.937H7.273a2.384,2.384,0,0,1-2.239-2.5V4.563a2.384,2.384,0,0,1,2.239-2.5h9.454a2.384,2.384,0,0,1,2.239,2.5V19.437A2.384,2.384,0,0,1,16.727,21.937ZM7.273,3.063a1.39,1.39,0,0,0-1.239,1.5V19.437a1.39,1.39,0,0,0,1.239,1.5h9.454a1.39,1.39,0,0,0,1.239-1.5V4.563a1.39,1.39,0,0,0-1.239-1.5Z"
                        />
                      </g>
                    </g>
                  </svg>
                </div>
                <div
                  class="about-feature-descriptions"
                >
                  <div
                    class="about-heading-4-wrapper"
                  >
                    <div
                      class="about-heading-4"
                    >
                      Mobile learning
                    </div>
                  </div>
                  <div
                    class="about-container"
                  >
                    <p
                      class="about-content"
                    >
                      Quisque nec volutpat sem. Interdum et malesuada fames ac ante ipsum primis in faucibus.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="about-feature-items"
              >
                <div
                  class="about-icon-parent"
                >
                  <svg
                    class="about-feature-icons"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 24 24"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M3.5 8a5.5 5.5 0 1 1 8.596 4.547 9.005 9.005 0 0 1 5.9 8.18.751.751 0 0 1-1.5.045 7.5 7.5 0 0 0-14.993 0 .75.75 0 0 1-1.499-.044 9.005 9.005 0 0 1 5.9-8.181A5.496 5.496 0 0 1 3.5 8ZM9 4a4 4 0 1 0 0 8 4 4 0 0 0 0-8Zm8.29 4c-.148 0-.292.01-.434.03a.75.75 0 1 1-.212-1.484 4.53 4.53 0 0 1 3.38 8.097 6.69 6.69 0 0 1 3.956 6.107.75.75 0 0 1-1.5 0 5.193 5.193 0 0 0-3.696-4.972l-.534-.16v-1.676l.41-.209A3.03 3.03 0 0 0 17.29 8Z"
                    />
                  </svg>
                </div>
                <div
                  class="about-feature-descriptions"
                >
                  <div
                    class="about-heading-4-wrapper"
                  >
                    <div
                      class="about-heading-4"
                    >
                      Academic & Technical Support
                    </div>
                  </div>
                  <div
                    class="about-container"
                  >
                    <p
                      class="about-content"
                    >
                      Quisque nec volutpat sem. Interdum et malesuada fames ac ante ipsum primis in faucibus.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="about-feature-items"
              >
                <div
                  class="about-icon-parent"
                >
                  <svg
                    class="about-feature-icons"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 32 32"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M 16 3 C 15.375 3 14.753906 3.210938 14.21875 3.59375 L 12.5625 4.75 L 10.65625 5 L 10.625 5 L 10.59375 5.03125 C 9.320313 5.316406 8.316406 6.320313 8.03125 7.59375 L 8 7.625 L 8 7.65625 L 7.75 9.59375 L 6.59375 11.09375 L 6.5625 11.125 L 6.5625 11.15625 C 5.863281 12.273438 5.832031 13.714844 6.59375 14.78125 L 7.78125 16.4375 L 8.09375 18.15625 L 4.875 23.0625 L 3.84375 24.59375 L 8.625 24.59375 L 9.78125 27.28125 L 10.5 29 L 11.53125 27.4375 L 14.6875 22.6875 C 15.535156 23.035156 16.492188 23.066406 17.3125 22.6875 L 20.46875 27.4375 L 21.5 29 L 22.21875 27.28125 L 23.375 24.59375 L 28.15625 24.59375 L 27.125 23.0625 L 24 18.3125 L 24.25 16.4375 L 25.40625 14.78125 L 25.4375 14.75 L 25.4375 14.71875 C 26.136719 13.601563 26.167969 12.191406 25.40625 11.125 L 24.25 9.46875 L 23.875 7.59375 L 23.90625 7.59375 C 23.902344 7.570313 23.878906 7.554688 23.875 7.53125 C 23.695313 6.222656 22.660156 5.160156 21.34375 5 L 21.3125 5 L 19.4375 4.75 L 17.78125 3.59375 C 17.246094 3.210938 16.625 3 16 3 Z M 16 5.03125 C 16.230469 5.03125 16.457031 5.101563 16.625 5.21875 L 18.40625 6.5 L 18.625 6.65625 L 18.875 6.6875 L 21.0625 7 L 21.09375 7 C 21.542969 7.050781 21.855469 7.363281 21.90625 7.8125 L 21.90625 7.875 L 22.3125 10.09375 L 22.34375 10.3125 L 22.5 10.5 L 23.78125 12.28125 C 24.019531 12.613281 24.050781 13.175781 23.75 13.65625 L 22.34375 15.625 L 22.3125 15.875 L 22 18.0625 L 22 18.09375 C 21.980469 18.257813 21.925781 18.410156 21.84375 18.53125 L 21.78125 18.5625 L 21.78125 18.59375 C 21.636719 18.765625 21.4375 18.878906 21.1875 18.90625 L 21.125 18.90625 L 18.84375 19.3125 L 18.59375 19.34375 L 18.40625 19.5 L 16.625 20.78125 C 16.292969 21.019531 15.699219 21.050781 15.21875 20.75 L 13.59375 19.5 L 13.40625 19.34375 L 13.125 19.3125 L 10.9375 19 L 10.90625 19 C 10.597656 18.964844 10.359375 18.804688 10.21875 18.5625 C 10.15625 18.453125 10.109375 18.324219 10.09375 18.1875 L 10.09375 18.125 L 9.6875 15.84375 L 9.65625 15.59375 L 9.5 15.40625 L 8.21875 13.625 C 7.980469 13.292969 7.949219 12.699219 8.25 12.21875 L 9.5 10.59375 L 9.65625 10.40625 L 9.6875 10.125 L 9.96875 8.03125 C 9.972656 8.015625 9.996094 8.015625 10 8 C 10.125 7.511719 10.511719 7.125 11 7 C 11.015625 6.996094 11.015625 6.972656 11.03125 6.96875 L 13.125 6.6875 L 13.375 6.65625 L 13.59375 6.5 L 15.375 5.21875 C 15.542969 5.101563 15.769531 5.03125 16 5.03125 Z M 22.90625 20.25 L 24.4375 22.59375 L 22.03125 22.59375 L 21.78125 23.21875 L 21.09375 24.8125 L 18.96875 21.5625 L 19.4375 21.21875 L 21.40625 20.875 L 21.40625 20.90625 C 21.429688 20.902344 21.445313 20.878906 21.46875 20.875 C 22.007813 20.800781 22.496094 20.574219 22.90625 20.25 Z M 9.09375 20.28125 C 9.519531 20.664063 10.0625 20.929688 10.65625 21 C 10.667969 21 10.675781 21 10.6875 21 L 12.59375 21.25 L 13.03125 21.59375 L 10.90625 24.8125 L 10.21875 23.21875 L 9.96875 22.59375 L 7.5625 22.59375 Z"
                    />
                  </svg>
                </div>
                <div
                  class="about-feature-descriptions"
                >
                  <div
                    class="about-heading-4-wrapper"
                  >
                    <div
                      class="about-heading-4"
                    >
                      Sharable Certificates
                    </div>
                  </div>
                  <div
                    class="about-container"
                  >
                    <p
                      class="about-content"
                    >
                      Quisque nec volutpat sem. Interdum et malesuada fames ac ante ipsum primis in faucibus.
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="about-feature-items"
              >
                <div
                  class="about-icon-parent"
                >
                  <svg
                    class="about-feature-icons"
                    fill="currentColor"
                    height="1em"
                    stroke="currentColor"
                    stroke-width="0"
                    viewBox="0 0 1024 1024"
                    width="1em"
                    xmlns="http://www.w3.org/2000/svg"
                  >
                    <path
                      d="M1025.02 512c0-272.016-213.663-495.104-482.319-511.023-5.536-.608-11.088-1.009-16.72-1.009-1.664 0-3.328.176-4.992.224-2.992-.048-5.968-.224-8.992-.224C229.117-.032-1.026 229.664-1.026 512s230.144 512.032 513.023 512.032c3.024 0 6-.176 9.008-.24 1.664.064 3.328.24 4.992.24 5.632 0 11.184-.4 16.72-1.009 268.64-15.92 482.304-238.976 482.303-511.023zm-95.451 164.832c-17.632-5.12-61.92-16.24-140.064-25.392 6.464-44.192 10-90.896 10-139.44 0-38.256-2.208-75.343-6.288-111.008 99.008-11.824 142.384-26.72 145.296-27.745l-11.92-33.584c22.24 53.088 34.56 111.296 34.56 172.336 0 58.193-11.28 113.761-31.583 164.833zM285.488 512.001c0-35.808 2.37-70.77 6.705-104.401 51.888 4.08 113.936 7.088 186.863 7.792v222.064c-70.992.688-131.664 3.568-182.688 7.473-7.04-42.193-10.88-86.88-10.88-132.928zM542.945 68.223c78.464 22.736 145.648 131.695 175.744 276.111-48.368 3.856-106.624 6.673-175.744 7.33V68.223zm-63.886.783V351.63c-68.368-.688-126.88-3.473-176.063-7.232C333.7 201.79 401.428 93.646 479.059 69.006zm0 632.223l.001 253.743c-72.4-22.976-136.192-118.575-169.36-247.023 47.76-3.504 104.096-6.063 169.359-6.72zm63.888 254.543l-.001-254.56c65.952.623 122.064 3.28 169.217 6.928-32.608 130.128-96 226.416-169.216 247.632zm-.001-318.32l.001-222.032c73.311-.688 134.991-3.776 186.191-8a844.922 844.922 0 0 1 6.496 104.592c0 46.128-3.712 90.864-10.528 133.12-50.416-4.08-110.8-7.008-182.16-7.68zm371.858-323.52c-9.664 3.008-50.063 14.48-131.023 24.032-18.048-95.952-50.672-177.968-93.12-237.168C788.197 143.18 867.797 219.1 914.805 313.932zM358.82 90.589c-52.208 59.952-94.832 146.161-118.096 248.113-72.48-7.856-115.921-17.089-133.312-21.281 50.72-104.64 141.04-186.752 251.408-226.832zM83.637 377.182c12.32 3.344 58.913 14.941 145.553 24.525a795.86 795.86 0 0 0-7.68 110.305c0 48.273 4.368 94.721 12.24 138.688-74.4 8.033-120.16 17.649-140.688 22.609-19.44-50.096-30.208-104.447-30.208-161.312 0-46.96 7.312-92.256 20.783-134.815zm37.457 355.166c23.264-4.944 64.912-12.464 126.592-18.928 24.288 89.712 63.792 165.616 111.136 219.968-101.12-36.72-185.296-108.752-237.728-201.04zM690.662 923.18c38.224-53.264 68.48-125.024 87.296-208.801 63.408 7.28 103.216 15.792 123.296 20.864-48.016 83.072-121.855 149.393-210.592 187.937z"
                    />
                  </svg>
                </div>
                <div
                  class="about-feature-descriptions"
                >
                  <div
                    class="about-heading-4-wrapper"
                  >
                    <div
                      class="about-heading-4"
                    >
                      An Inclusive Experience
                    </div>
                  </div>
                  <div
                    class="about-container"
                  >
                    <p
                      class="about-content"
                    >
                      Quisque nec volutpat sem. Interdum et malesuada fames ac ante ipsum primis in faucibus.
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="about-our-story"
        >
          <div
            class="about-story-content"
          >
            <div
              class="about-story-wrapper"
            >
              <div
                class="about-heading-2-our-story"
              >
                <h2
                  class="about-heading-23"
                >
                  Our Story
                </h2>
                <div
                  class="about-line-story"
                />
              </div>
              <div
                class="about-our-story-container"
              >
                <p
                  class="about-our-story-content"
                >
                  Lorem ipsum dolor sit amet, consectetur adipiscing elit. Sed consectetur vel dolor id ultrices. Proin Link magna at mi pretium pulvinar in eu enim. Nulla vel lacus lectus. Donec at venenatis augue. Nam vitae purus placerat, hendrerit nisl id, finibus magna. Etiam pharetra gravida ornare. Donec sagittis, ipsum in egestas egestas, dui lorem sollicitudin justo, ut ullamcorper velit neque eu velit. Ut et fringilla elit. Mauris augue augue, auctor Link blandit ac, convallis eget neque. Curabitur in ante ante. Nullam laoreet tempus erat at ornare. In nisl nisi, dapibus eget facilisis sit amet, commodo quis nibh.
                </p>
              </div>
            </div>
          </div>
          <img
            alt=""
            class="about-story-img"
            src="https://gambolthemes.net/html-items/cursus-new-demo/images/about/stroy_img.png"
          />
        </div>
        <div
          class="about-features-content-wrapper-our-global"
        >
          <div
            class="about-features-content"
          >
            <div
              class="about-features-wrapper"
            >
              <div
                class="about-features-inner-wrapper"
              >
                <div
                  class="about-heading-2-our-features"
                >
                  <h2
                    class="about-heading-22"
                  >
                    Our Global Reach
                  </h2>
                </div>
                <div
                  class="about-our-global-parent"
                >
                  <div
                    class="about-on-cursus"
                  >
                    Cursus is the leading global marketplace for teaching and learning, connecting millions of students to the skills they need to succeed.
                  </div>
                  <div
                    class="about-line-wrapper"
                  />
                </div>
              </div>
            </div>
            <div
              class="about-feature-list-our-global"
            >
              <div
                class="about-feature-items-our-global"
              >
                <div
                  class="about-feature-descriptions-our-global"
                >
                  <div
                    class="about-heading-5-wrapper-our-global"
                  >
                    <h1
                      class="about-heading-5"
                    >
                      25K
                    </h1>
                  </div>
                  <div
                    class="about-container-our-global"
                  >
                    <p
                      class="about-content-our-global"
                    >
                      Instructors
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="about-feature-items-our-global"
              >
                <div
                  class="about-feature-descriptions-our-global"
                >
                  <div
                    class="about-heading-5-wrapper-our-global"
                  >
                    <h1
                      class="about-heading-5"
                    >
                      95K
                    </h1>
                  </div>
                  <div
                    class="about-container-our-global"
                  >
                    <p
                      class="about-content-our-global"
                    >
                      Course
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="about-feature-items-our-global"
              >
                <div
                  class="about-feature-descriptions-our-global"
                >
                  <div
                    class="about-heading-5-wrapper-our-global"
                  >
                    <h1
                      class="about-heading-5"
                    >
                      40M
                    </h1>
                  </div>
                  <div
                    class="about-container-our-global"
                  >
                    <p
                      class="about-content-our-global"
                    >
                      Course enrollments
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="about-feature-items-our-global"
              >
                <div
                  class="about-feature-descriptions-our-global"
                >
                  <div
                    class="about-heading-5-wrapper-our-global"
                  >
                    <h1
                      class="about-heading-5"
                    >
                      50+
                    </h1>
                  </div>
                  <div
                    class="about-container-our-global"
                  >
                    <p
                      class="about-content-our-global"
                    >
                      Languages
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="about-feature-items-our-global"
              >
                <div
                  class="about-feature-descriptions-our-global"
                >
                  <div
                    class="about-heading-5-wrapper-our-global"
                  >
                    <h1
                      class="about-heading-5"
                    >
                      595+
                    </h1>
                  </div>
                  <div
                    class="about-container-our-global"
                  >
                    <p
                      class="about-content-our-global"
                    >
                      Membership Partner
                    </p>
                  </div>
                </div>
              </div>
              <div
                class="about-feature-items-our-global"
              >
                <div
                  class="about-feature-descriptions-our-global"
                >
                  <div
                    class="about-heading-5-wrapper-our-global"
                  >
                    <h1
                      class="about-heading-5"
                    >
                      295
                    </h1>
                  </div>
                  <div
                    class="about-container-our-global"
                  >
                    <p
                      class="about-content-our-global"
                    >
                      Countries
                    </p>
                  </div>
                </div>
              </div>
            </div>
          </div>
        </div>
        <div
          class="about-meet-our-team"
        >
          <div
            class="about-team-content"
          >
            <div
              class="about-team-wrapper"
            >
              <div
                class="about-heading-6-wrapper"
              >
                <h3
                  class="about-heading-6"
                >
                  Meet Our Team
                </h3>
              </div>
              <div
                class="about-meet-parent"
              >
                <div
                  class="about-meet-container"
                >
                  A perfect blend of creativity and technical wizardry. The best people formula for great websites!
                </div>
                <div
                  class="about-line-wrappers"
                />
              </div>
            </div>
          </div>
          <div
            class="about-team-call-to-action-parent"
          >
            <div
              class="about-team-call-to-action"
            >
              <div
                class="about-team-call-container"
              >
                <p
                  class="about-team-call-content"
                >
                  Morbi eget elit eget turpis varius mollis eget vel massa. Donec porttitor, sapien eget commodo vulputate, erat felis aliquam dolor, non condimentum libero dolor vel ipsum. Sed porttitor nisi eget nulla ullamcorper eleifend. Fusce tristique sapien nisi, vel feugiat neque luctus sit amet. Quisque consequat quis turpis in mattis. Maecenas eget mollis nisl. Cras porta dapibus est, quis malesuada ex iaculis at. Vestibulum egestas tortor in urna tempor, in fermentum lectus bibendum. In leo leo, bibendum at pharetra at, tincidunt in nulla. In vel malesuada nulla, sed tincidunt neque. Phasellus at massa vel sem aliquet sodales non in magna. Ut tempus ipsum sagittis neque cursus euismod. Vivamus luctus elementum tortor, ac aliquet dolor vehicula et.
                </p>
              </div>
              <button
                class="about-button-join-our-team"
              >
                Join Our Team
              </button>
            </div>
            <img
              alt=""
              class="about-team-img"
              src="https://gambolthemes.net/html-items/cursus-new-demo/images/about/team.jpg"
            />
          </div>
        </div>
      </div>
    </main>
    <footer
      class="footer footer-large-container"
    >
      <div
        class="footer-links"
      >
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/about-us"
          >
            About
          </a>
          <a
            class="link"
            href="/blog-single-view/:id"
          >
            Blog
          </a>
          <a
            class="link"
            href="/career"
          >
            Careers
          </a>
          <a
            class="link"
            href="/press"
          >
            Press
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/help"
          >
            Help
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Advertise
          </a>
          <a
            class="link"
            href="/coming-soon"
          >
            Developers
          </a>
          <a
            class="link"
            href="/contactus"
          >
            Contact Us
          </a>
        </div>
        <div
          class="footer-column"
        >
          <a
            class="link"
            href="/termsofuse"
          >
            Copyright Policy
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Terms
          </a>
          <a
            class="link"
            href="/termsofuse"
          >
            Privacy Policy
          </a>
          <a
            class="link"
            href="/sitemap"
          >
            Sitemap
          </a>
        </div>
        <div
          class="footer-column"
        >
          <button
            class="teach-button"
          >
            Teach on Cursus
          </button>
          <div
            class="language-selector"
          >
            <button>
              Language 
              <span>
                ▼
              </span>
            </button>
          </div>
        </div>
      </div>
      <div
        class="footer-bottom"
      >
        <div
          class="footer-copy"
        >
          <img
            alt=""
            src="https://gambolthemes.net/html-items/cursus-new-demo/images/logo1.svg"
          />
          © 2024 
          <strong>
            Cursus
          </strong>
          . All Rights Reserved.
        </div>
        <div
          class="footer-social"
        >
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="facebook.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="x.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="google+.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="linkedin.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="insta.png"
            />
          </a>
          <a
            href="/home"
          >
            <img
              alt=""
              class="social-icon"
              src="pinterest.png"
            />
          </a>
        </div>
      </div>
    </footer>
  </div>
</div>
`;
